unfoldingWord/translationCore
# Added React 16

#### This pull request addresses:
This PR changes dependencies to enable react 16


#### How to test this pull request:
Make sure tests pass and the app functions as normal


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3626)
